source/fundamentals/linq.txt

```diff
@@ -0,0 +1,531 @@
+``reviews`` collection with a ``name`` field that matches the ``name`` of the 
+document from the ``restaurants`` collection. 
+
+The following is a subset of the returned results:
+
+.. code-block:: json
+
+   // Results Truncated
+
+   { "restaurant" : {
+      "_id" : ObjectId("..."),
+      "name" : "The Movable Feast", 
+      "restaurant_id" : "40361606", 
+      "cuisine" : "American", 
+      "address" : {...}, 
+      "borough" : "Brooklyn", 
+      "grades" : [...] }, 
+      "reviews" : [
+        { "_id" : ObjectId(...), "restaurant_name" : "The Movable Feast", "reviewer" : "Lazlo Cravensworth", "review" : "Great restaurant! 12/10 stars!" },
+        { "_id" : ObjectId("..."), "restaurant_name" : "The Movable Feast", "reviewer" : "Michael Scarn", "review" : "It really was a feast" }
+       ]
+    }
+
+Unsupported Aggregation Stages
+------------------------------
+
+The {+driver-long+} implementation of LINQ does not support the following
+aggregation stages:
+
+- ``$redact``
+- ``$geoNear``
+- ``$out``
+
+Supported Methods
+-----------------
+
+The {+driver-long+} implementation of LINQ supports the following methods:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Method Name
+     - Description
+   
+   * - ``Any``
+     - Determines if any documents match the specified criteria
+
+   * - ``Average``
+     - Calculates the average of the specified fields
+
+   * - ``Count``
+     - Returns the number of documents that match the specified criteria
+
+   * - ``LongCount``
+     - Returns an ``Int64`` that represents the number of documents that match the specified criteria
+
+   * - ``Distinct``
+     - Returns distinct documents that match the specified criteria
+
+   * - ``First``
+     - Returns the first matching document
+
+   * - ``FirstOrDefault``
+     - Returns the first matching document, or ``null`` if none are found
+
+   * - ``GroupBy``
+     - Groups documents based on specified criteria
+
+   * - ``GroupJoin``
+     - Performs a left outer join to another collection in the same database
+
+   * - ``Max``
+     - Returns the document with the maximum specified value
+
+   * - ``OfType``
+     - Returns documents that match the specified of type
+
+   * - ``OrderBy``, ``OrderByDescending``
+     - Returns results in a specified sort order
+
+   * - ``ThenBy``, ``ThenByDescending``
+     - Allows a secondary sort to be specified
+
+   * - ``Select``
+     - Selects documents based on specified criteria
+
+   * - ``SelectMany``
+     - Projects each element of a sequence and combines the resulting sequences into one document.
+
+   * - ``Single``
+     - Returns a single matching document
+   
+   * - ``SingleOrDefault``
+     - Returns a single matching document or ``null`` if no documents match
+
+   * - ``Skip``
+     - Skips over a specified number of documents and returns the rest of the results
+
+   * - ``Sum``
+     - Returns the sum of the values in specified fields
```

Comment:
Can this sum multiple fields? If not, should this be "field", singular?

Reply:
Changed to singular